.github/workflows/docker-nightly-build.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image Nightly Build CI
+
+on:
+  push:
+    branches: [ cron_build ]
```

Comment:
No need to keep this after the experiment is done.

Reply:
Isn't it still useful to have this to debug any job failures? That's the reason I kept it. No strong opinion either way.